person.rb

```diff
@@ -17,5 +18,9 @@ def can_use_services?
     age >= 18 || parent_permission == true
   end
 
+  def validate_name
+    @name = @corrector.correct_name
```

Comment:
- [ ] You should pass `@name` to the `correct_name` method.

`@name = @corrector.correct_name(@name)`